docs/src/helpers/use-preset-highlight.ts

```diff
@@ -1,21 +1,24 @@
 import { useEffect } from 'react';
 
-export const usePresetHighlight = (presets: Record<string, any>[], params: Record<string, any>) => {
+export const usePresetHighlight = (presets: Record<string, any>[], levaParams: Record<string, any>) => {
   useEffect(() => {
-    // Leva takes a little longer to mount on some examples, so buttons are not present at the time of the
-    // first render. Delaying this hook with a timeout, even if it's 0, ensures that the buttons are present.
-    const timeoutId = setTimeout(() => {
+    const highlightPreset = () => {
       const matchingPreset = presets.find((preset) => {
-        // Remove anything present in the preset that is not a param
-        const { seed, ...rest } = preset.params;
+        // We might need to remove properties present in the preset that are not changeable params in the future
```

Comment:
I think I added this bit because it was causing it to crash when `seed` was provided by not expected.

Reply:
I removed it because I thought all presets now had `seed` in Leva, but I now see that it's not always the case (dots grid doesn't have it, for example). I'll put it back 👍

Reply:
Yeah I'd actually like to not have `seed` in Leva unless it's a non-animated shader. It just seems to confuse folks.